src/region.js

```diff
@@ -269,7 +275,11 @@ const Region = MarionetteObject.extend({
     delete this.currentView;
 
     if (!view._isDestroyed) {
-      this._removeView(view, shouldDestroy);
+      if (shouldDestroy) {
+        this.removeView(view);
```

Comment:
To make easier to override, without `destroyView`, with its content inside `removeView` would be necessary to do something like below to accommodate standalone BB.View:
```
removeView(view) {
    view.$el.fadeOut('slow', () => {
          if (view.destroy) {
            view.destroy();
           } else {
             destroyBackboneView(view);
            }
        this.currentView.$el.fadeIn('slow')
    })    
  } 
```
The problem is that `destroyBackboneView` is not exposed

or using ES6 super

```
removeView(view) {
    view.$el.fadeOut('slow', () => {
        this.super.removeView(view)
        this.currentView.$el.fadeIn('slow')
    })    
  } 
```

Anyway, as i said the destroyView is optional and can be removed